index.html

```diff
@@ -762,18 +839,137 @@ <h2><dfn>CaptureController</dfn></h2>
+            </dt>
             <dd>
               <p>
-                Create one internal slot: <dfn data-dfn-for="CaptureController">[[\isBound]]</dfn>,
-                initialized to <code>false</code>.
+                Run the following steps:
               </p>
+              <ol>
+                <li>
+                  <p>
+                    Let <var>focusBehavior</var> be the method's first argument.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    If {{CaptureController/[[isBound]]}} is <code>false</code>,
+                    [=exception/throw=] an "{{InvalidStateError}}" {{DOMException}}.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    If {{CaptureController/[[videoTrack]]}} is <code>null</code>,
+                    [=exception/throw=] a "{{InvalidStateError}}" {{DOMException}}.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    If {{CaptureController/[[videoTrack]]}}.{{MediaStreamTrack/getSettings()}}.{{displaySurface}}
+                    is neither {{DisplayCaptureSurfaceType/"browser"}} nor {{DisplayCaptureSurfaceType/"window"}},
+                    [=exception/throw=] a "{{NotAllowedError}}" {{DOMException}}.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    <!-- TODO: Fix the ReadyState reference. -->
+                    If {{CaptureController/[[videoTrack]]}}'s
+                    <code>ReadyState</code> is NOT {{MediaStreamTrackState/"live"}},
+                    [=exception/throw=] a "{{NotAllowedError}}" {{DOMException}}.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    If {{CaptureController/[[focusWindowClosed]]}} is <code>true</code>,
+                    [=exception/throw=] a "{{NotAllowedError}}" {{DOMException}}.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    If {{CaptureController/[[initialFocusLost]]}} is <code>true</code>,
+                    abort these steps.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    The user agent SHOULD abort these steps if too much time has elapsed since
+                    the [=capture-session=] started. The timespan is left up to the user agent,
+                    but it is recommended that a value of one second be used.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    Set {{CaptureController/[[focusWindowClosed]]}} to <code>true</code>.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    If <var>focusBehavior</var> is NOT {{CaptureStartFocusBehavior/"focus-captured-surface"}},
```

Comment:
Shouldn't we specify that the display surface MUST NOT be focused  when "no-focus-change". Or at least a MAY NOT be focused.